components/ledger/ledger-utxo-token-cache/src/main/kotlin/net/corda/ledger/utxo/token/cache/entities/internal/TokenBalanceImpl.kt

```diff
@@ -0,0 +1,14 @@
+package net.corda.ledger.utxo.token.cache.entities.internal
+
+import net.corda.v5.ledger.utxo.token.selection.TokenBalance
+import java.math.BigDecimal
+
+internal class TokenBalanceImpl(private val availableBalance: BigDecimal, private val totalBalance: BigDecimal) : TokenBalance {
```

Comment:
I would suggest re-naming:
```suggestion
internal class TokenBalanceCacheImpl(private val availableBalance: BigDecimal, private val totalBalance: BigDecimal) : TokenBalance {
```
just to avoid name clash with existing `TokenBalanceImpl` from a different module.